docs/privileged-identity-management-requests.md

```diff
@@ -0,0 +1,19 @@
+# Privileged Identity Management (PIM) requests
+
+Accessing resources in the `production` or `test` environments requires elevated
```

Comment:
Isn't the environment called `development`?

```suggestion
Accessing resources in the `production` or `development` environments requires elevated
```

Reply:
`development` doesn’t need PIM, AFAIK. Only `production` and for some reason `test`.